src/components/PostList/PostList.scss

```diff
@@ -1 +1 @@
-// add styles here
+// add styles here
```

Comment:
The `PostList.scss` file is empty. If you intend to style the `PostList` component, you should add the necessary CSS rules here.